content/cybersecurity/concepts/security-testing.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Please note that the metadata should look like this:

```
---
Title: 'The Title'
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
Tags:
  - 'A tag'
  - 'A second tag'
  - 'An nth tag'
CatalogContent:
  - 'learn-example-course'
  - 'path/example-path'
---
```
Note the lack of line breaks and the `---` at the end.

Also tags should come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).
And subjects should come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).